### PR TITLE
Expose VPC id 

### PR DESCRIPTION
**Pull request by dotemacs**
Exposing the VPC id so that they can be accessed upstream

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -113,3 +113,8 @@ output "elb.masters_internal_dns_name" {
   description = "This is the load balancer to access the masters internally in the cluster"
   value       = "${module.dcos-elb.masters_internal_dns_name}"
 }
+
+output "vpc.id" {
+  description = "This is the id of the VPC the cluster is in"
```
Comment (fatz) [inline outputs.tf:118]: @geekbass any thought on the description. Think its ok and I'll put it into the description doc

Comment (geekbass) [reply outputs.tf:118]: Yes I think this is a sufficient description. Im good with it!

**Review by geekbass — APPROVED** [review]
LGTM!

**Comment by fatz** [comment]
@dotemacs our jenkins job is using a specific version of `terraform-docs` to proof if the README.md is correctly generated. Can you please run 

```
docker run -ti -v$(pwd):/a --entrypoint /usr/bin/terraform-docs fatz/terraform-jenkins md /a > README.md
```

in your workdir and commit the change. It looks like there is a newline missing.

**Review by fatz — DISMISSED** [review]
see my comment.

**Review by geekbass — APPROVED** [review]
👍

**Comment by fatz** [comment]
thx @dotemacs we took your input and implemented it in #10

**Pull request closed without merging**